src/renderer/containers/app.css

```diff
@@ -52,3 +52,34 @@ span.clickable {
 ::-webkit-scrollbar { width:8px; height:8px }
 ::-webkit-scrollbar-track { background: #fff; }
 ::-webkit-scrollbar-thumb { background: #666; border-radius:5px }
+
+
+/* Dark theme */
+body.dark-theme {
+  filter: invert(100%);
+  background-color: #333;
+}
+
+body.dark-theme .Select-control,
+body.dark-theme .ui.form input:not([type]),
+body.dark-theme .ui.form input[type="date"],
+body.dark-theme .ui.form input[type="datetime-local"],
+body.dark-theme .ui.form input[type="email"],
+body.dark-theme .ui.form input[type="number"],
+body.dark-theme .ui.form input[type="password"],
+body.dark-theme .ui.form input[type="search"],
+body.dark-theme .ui.form input[type="tel"],
+body.dark-theme .ui.form input[type="time"],
+body.dark-theme .ui.form input[type="text"],
+body.dark-theme .ui.form input[type="url"] {
+  border-color: #000;
+}
+body.dark-theme .ui.toggle.checkbox .box:before,
+body.dark-theme .ui.toggle.checkbox label:before {
+  border: 1px solid #333;
+  background: #ccc;
+}
+body.dark-theme .ui.toggle.checkbox .box:after,
+body.dark-theme .ui.toggle.checkbox label:after {
+  box-shadow: 0px 1px 2px 0 #333, 0px 0px 0px 1px #333 inset;
+}
```

Comment:
This is the main style for dark theme. Due to some issues with contrast in the Toggle Checkbox, I need to add some outlines here.

Reply:
Also I upgraded Semantic UI css to the latest